### PR TITLE
Fixing and Improving InfiniteGlass Docker container

### PR DESCRIPTION
**Pull request by BackOrder**
- Dockerfile rewritten to improve performances
  - Packages are grouped, i.e. Python packages, X11 packages, etc.
  - Better practices, ARGS/ENV on top, packages less likely to be
    modified, and using ENTRYPOINT rather than CMD.
- Removed faulty PyGObject in `glass-action/setup.py`.

### CODE REVIEW

**Comment by BackOrder** [comment]
Ref. issue #71

File: glass-action/setup.py [hunk 1]
```diff
@@ -13,7 +13,6 @@
       packages=setuptools.find_packages(),
       install_requires=[
           "click",
-          "PyGObject"
```
Comment (redhog) [inline glass-action/setup.py:16]: Why did you remove this? Ideally all pip requirements should be installed by the setup.py:s, not Dockerfile, or stuff won't work _outside_ of docker.

Comment (BackOrder) [reply glass-action/setup.py]: Done.
